cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/AbstractOutputsTransformer.java

```diff
@@ -33,7 +33,9 @@
  * Date: 12/11/2014
  * Time: 11:33
  */
-public class AbstractOutputsTransformer  extends InOutTransformer {
+public abstract class AbstractOutputsTransformer  extends InOutTransformer {
+
+    public static final List<String> KNOWN_KEYS = Arrays.asList(SENSITIVE_KEY, VALUE_KEY);
```

Comment:
This list of keys should be moved to the OutputTransformer. It is relevnt only there